pipelines/templates/tasks/upmpackages.yml

```diff
@@ -2,11 +2,12 @@
 parameters:
   version: $(MRTKVersion)
   buildNumber: $(Build.BuildNumber)
+  publishUpm: $(PublishUpmPackages)
```

Comment:
Should `PublishUpmPackages` be added to [our settings file](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/pipelines/config/settings.yml) so it's checked in? Where is it defined?

Reply:
It is defined in the pipeline varaiables

Reply:
the same is true for $(UpmRegistry)

Reply:
We might want to explicitly pass the parameter in via the pipeline tasks too, so it's clear which pipeline will publish or not by looking at the yaml.

Reply:
updated